srcCpp/Infrastructure_common.h

```diff
@@ -79,4 +222,17 @@ class PerftestFileHandler
             unsigned int startPos = 0);
 };
 
+/*
+ * Since std::to_string is not defined until c++11
+ * we will define it here.
+ */
+namespace std {
+    template<typename T>
+    std::string to_string(const T &n) {
```

Comment:
I haven't seen issues when compiling for CycloneDDS and FastDDS and they enforce C++11 in the CMake file.

Is there a way to enable this exclusively when compiling with no C++11?

Reply:
Just define this function in a different namespace and use it instead of `std::to_string`.

Reply:
It might work because a `to_string` definition with a concrete type will take precedence over a template during overload resolution. But this can break if the standard library had some kind of templated `to_string`.

Reply:
I see. What about perftest::to_string? That should be enough for the time being, right?

Reply:
Done.